core/controllers/concept_card_viewer.py

```diff
@@ -15,26 +15,26 @@
 """Controllers for the Oppia skill's concept card viewer."""
 
 from core.controllers import base
+from core.domain import acl_decorators
 from core.domain import skill_services
 import feconf
 
 
 class ConceptCardDataHandler(base.BaseHandler):
     """A card that shows the explanation of a skill's concept."""
 
+    @acl_decorators.can_play_skill
     def get(self, skill_id):
         """Handles GET requests."""
 
         if not feconf.ENABLE_NEW_STRUCTURES:
             raise self.PageNotFoundException
 
         skill = skill_services.get_skill_by_id(skill_id, strict=False)
-        if skill is None:
```

Comment:
Should keep the guard if we end up subsuming the decorator into e.g. can_play_exploration.

Reply:
Yeah, I'll move this to the actual method. I thought once skill publishing is done, whether a skill is published could be checked here.